pyfranca/franca_processor.py

```diff
@@ -397,6 +403,16 @@ def import_file(self, fspec, references=None, package_path=None):
                 else:
                     raise ProcessorException(
                         "Model '{}' not found.".format(fspec))
+
+            if not withError:
```

Comment:
This change breaks a few valid test cases:

- test_import_nonexistent_namespace
- test_package_in_multiple_files_reuse
- test_interface_visibility
- test_interface_extension2
- test_reference_to_different_model
- test_reference_priority